### ISSUE TITLE
Summer / Winter time Bug

### ISSUE THREAD

**Issue by MarkyMarkDE** [d4rken-org/sdmaid-se#768]
When the time change from summer to winter or winter to summer (UTC +0100 or UTC +0200), SDMaid did not recognize the new time, it stays at the previous one and the scheduled job starts wrong. 
In system the time is correct at this time.
This bug exists on Pixel 7 and AndroidTV as well.

sdmaid-se-0.15.3-beta0-foss

**Comment by d4rken** [comment]
Timezones :dizzy_face:

**Comment by MarkyMarkDE** [comment]
> Timezones 😵

indeed, problem with the changed timezones, maybe this is casing in germany only

**Comment by MarkyMarkDE** [comment]
Indeed, this "bug" is still existing. Here on my Linux I must do a work around the keep my Cronjobs being executed at a static hour, equal if it is summer or winter time.
